docs/reference/setup/bootstrap-checks.asciidoc

```diff
@@ -130,6 +130,19 @@ address space. This can be done via `/etc/security/limits.conf` using
 the `as` setting to `unlimited` (note that you might have to increase
 the limits for the `root` user too).
 
+=== Max file size check
+
+The segment files that are the components of individual shards and the translog
+generations that are components of the translog can get large (on the order of
+hundreds of megabytes and even extending into gigabytes). On systems where the
```

Comment:
You're right, that was not intended, I pushed ee3580814c2f20c49983563c5955bb716a88891b.